composables/useMetaTransaction.ts

```diff
@@ -28,6 +29,7 @@ function useMetaTransaction() {
     onError?: () => void
   ): Promise<void> => {
     try {
+      isLoading.value = true
```

Comment:
How about using `initTransactionLoader` outside  the composables?

Reply:
I just switch to your branch to try to reproduce it. However, I remove this line `isLoading.value = true` and add `initTransactionLoader()` before `howAboutToExecute(xxx)`. The loader would show up. How about try again?

<img width="804" alt="image" src="https://user-images.githubusercontent.com/31397967/211576307-ab2ea344-4012-4b0d-8904-b416f2bd0c7c.png">

<img width="413" alt="image" src="https://user-images.githubusercontent.com/31397967/211575959-991017a7-e3fe-4384-90b0-df9a6c80e4bc.png">